backend/api/users.py

```diff
@@ -144,11 +141,8 @@ def delete_bookmarks(id):
 @users.route('/api/users/bookmarks', methods=['GET'])
 @login_required
 def get_bookmarks():
-    user = current_user
-    if user is None:
-        return make_response("No current user detected ", 400)
-    res = g.user_datastore.get_user(user.email)
-    if res is None:
+    res = g.user_datastore.get_user(current_user.email.to_python())
+    if not res:
         return make_response("Unknown User with Email-address: " +
```

Comment:
I'm not even sure if this makes sense. It's the current_user anyway and this is @login_required so there is literally no chance that the user is unknown. If for any reason the user_datastore can't find the user (?) this should raise an internal server error / apiexception.

Other input is welcome, just my thought.